bank_integration/bank_integration/api/hdfc_bank_api.py

```
@@ -36,98 +36,154 @@ def login(self):
+                            By.XPATH,
+                            '//button[contains(@class, "bb-button-bar__button") and contains(@class, "btn-primary") and normalize-space(text())="Get OTP"]',
+                        )
+                    ),
+                    EC.presence_of_element_located((By.ID, "mfa-get-otp-btn")),
+                ),
                 throw=False,
-            ).text
-        except NoSuchElementException:
-            pass
+            )
+        except:
```

Ruff (0.15.4)

[error] Do not use bare `except`

(E722)